units/src/amount.rs

```diff
@@ -1157,6 +1157,11 @@ impl ops::DivAssign<u64> for Amount {
 impl FromStr for Amount {
     type Err = ParseError;
 
+    /// Parses a string slice where the slice includes a denomination.
+    /// If the string slice is zero, then no denomination is required.
+    ///
+    /// Returns `Ok(Amount)` if the string amount and denomination parse
+    /// successfully, otherwise, return `Err(ParseDenominationError)`.
```

Comment:
Doesn't look like these docs are totally correct, return type is `ParseError` not `ParseDenominationError`.

Also, I rekon we should especially pay attention to polishing these here because `units` is on its way to stable ... hopefully. Please use `Returns` section and put space after first line of docs. (I see the rest of the module docs need work too :(

Reply:
Thanks for catching this.  Not sure how I manged to mess up the error message but it should be fixed.  Also added a Returns section.